tests/unit/test_paginate.py

```diff
@@ -334,6 +344,35 @@ def test_s3_list_multipart_uploads(self):
              ])
 
 
+class TestNonStringTokens(unittest.TestCase):
+    def setUp(self):
+        self.method = mock.Mock()
+        # This is something we'd see in s3 pagination.
+        self.paginate_config = {
+            "input_token": ["ExclusiveStartKey"],
+            "output_token": ["LastEvaluatedKey"],
+            "result_key": 'Items',
+        }
+        session = botocore.session.get_session()
+        service_model = session.get_service_model('dynamodb')
```

Comment:
How hard would it be to create your own model? It is nice to be able to avoid relying on service models so tests do not start failing when we update a model.